thirdparty/github.go

```diff
@@ -274,19 +298,49 @@ func GetGithubCommits(ctx context.Context, oauthToken, owner, repo, ref string,
 		options.Until = until
 	}
 
-	commits, resp, err := client.Repositories.ListCommits(ctx, owner, repo, &options)
-	if resp != nil {
-		defer resp.Body.Close()
+	installationToken, _ := GetInstallationToken(ctx, owner, repo, nil)
+	if installationToken != "" {
```

Comment:
Shouldn't we check the error here? Otherwise why are we even  returning an error?